public/app/core/hooks/useQueryParams.ts

```diff
@@ -6,9 +6,6 @@ import { useLocation } from 'react-router-dom';
 export function useQueryParams(): [UrlQueryMap, (values: UrlQueryMap, replace?: boolean) => void] {
   const { search } = useLocation();
   const queryParams = useMemo(() => locationSearchToObject(search || ''), [search]);
-  const update = useCallback(
-    (values: UrlQueryMap, replace?: boolean) => setImmediate(() => locationService.partial(values, replace)),
```

Comment:
idk why we're using `setImmediate`, but i don't think we should be: https://developer.mozilla.org/en-US/docs/Web/API/Window/setImmediate#browser_compatibility